### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by robertorodriguez12**
Updated the README.md file to incorporate a description of the project, how we developed and how we tested. Also added a visual representation of the project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,45 @@
...
+
+
+## Testing
+
+
+Testing was conducted with different approaches. For the development of our
+methods, we utilized fixtures, the fixtures were created by importing data
+from the Game, GameTeams and Teams CSV files. This allowed our testing to
+perform its basic function which is to determine if the method gave us the
+information we needed. Using fixture files also allowed the testint times to
```
Comment (ShaunDaneJames) [inline README.md]: testing*

**Review by ShaunDaneJames — CHANGES_REQUESTED** [review]
Great job Roberto! Not a big deal but spell check on line 30.

File: README.md (hunk 1 above)
Comment (robertorodriguez12) [reply README.md]: fixed the spelling. Let me know if you find any other typos.

**Review by ShaunDaneJames — APPROVED** [review]
Thanks Roberto, looks good to me.

**Review by dcoleman21 — COMMENTED** [review]
This looks really good Roberto.

**Pull request merged**